src/main/io/serial.h

```diff
@@ -51,6 +51,7 @@ typedef enum {
     FUNCTION_LIDAR_TF            = (1 << 15), // 32768
     FUNCTION_FRSKY_OSD           = (1 << 16), // 65536
     FUNCTION_VTX_MSP             = (1 << 17), // 131072
+    FUNCTION_GIMBAL              = (1 << 18), // 262144
 } serialPortFunction_e;
```

Comment:
(this way of configuration was acceptable with few bits. Some refactoring would be great)

Reply:
Another PR I think.